Makefile

```diff
@@ -37,9 +37,9 @@ endif
 
 APP_DEPLOYER_IMAGE ?= $(REGISTRY_PREFIX)/deployer:$(TAG)
 CONJUR_IMAGE ?= $(REGISTRY)/$(PREFIX):$(TAG)
-POSTGRES_SOURCE_IMAGE ?= postgres:10.1
+POSTGRES_SOURCE_IMAGE ?= postgres:12.2
 POSTGRES_IMAGE ?= $(REGISTRY_PREFIX)/postgres:$(TAG)
-NGINX_SOURCE_IMAGE ?= nginx:1.15
+NGINX_SOURCE_IMAGE ?= nginx:1.17.9
```

Comment:
this one I'm not sure about

the quick start (docker-compose deploy method for oss) uses 9.4 for postgres and 1.13.6 for nginx:
https://github.com/cyberark/conjur-quickstart/blob/8f8155452e82bcb0aaf331cb1b5a36957762f309/docker-compose.yml#L35-L53

the conjur aws repo (for cloudformation templates / AMIs) uses pg 9.3.17:
https://github.com/cyberark/conjur-aws/blob/3813e338889312a074587c7df3a17fce948bcadb/templates/cfn/conjur.yml.j2#L136

the conjur repo dev scripts use pg 9.4 and no nginx (not a good practice for real life, though):
https://github.com/cyberark/conjur/blob/10b1c78646e5a572d78af832d025c8944acd4e5d/dev/docker-compose.yml#L4

container-dap uses postgres 9.4 and nginx 1.16.1 (from a binary, so not sure if that maps to the docker image exaactly)
https://github.com/conjurinc/container-dap/blob/2582de4ea67bdd2c5876205689e22405bc3f84f4/images/postgres/Dockerfile#L23-L33
https://github.com/conjurinc/container-dap/blob/2582de4ea67bdd2c5876205689e22405bc3f84f4/NOTICES.txt#L24

can the @cyberark/conjur-core-team comment on whether the oss needs to have postgres pinned at 9.4, and if it matters what nginx is pinned at?

Reply:
NGINX is fine in most cases but I don't think we want to pin to the patch version so 1.17 would be good. If we can use an alpine-based image, that would be even better but that may require some more work in the upstream chart.

Reply:
Note: on our call today, we talked about:
- pinning postgres to 9.4
- using the container-dap postgres image if the dockerhub image has too many vulns
- pinning nginx to use the same major version as elsewhere but being more flexible about minor/patch since the nginx API is pretty stable

Reply:
So how about this?:
* Pin to postgres:9.4. This will make the marketplace app consistent/in-sync with cyberark/conjur. There are only a couple of critical vulnerabilities with postgres:9.4. 
* Upgrade nginx from 1.15 to 1.17. This minor version bump eliminates on critical vulnerability.